storage/redis/peer_store.go

```diff
@@ -810,6 +814,102 @@ func (ps *peerStore) collectGarbage(cutoff time.Time) error {
 	return nil
 }
 
+// Clear clears (almost) all data from redis, given that no concurrent
+// operations are taking place.
+// This uses the same logic as collectGarbage, only that all peers,
+// regardless of age, will be deleted.
+// In the end, only the counter keys remain.
+func (ps *peerStore) Clear() error {
```

Comment:
Redis has a concept called databases that are similar to schemas in Postgres or MySQL. We could just create a new database and switch to it. Then we don't have to worry about any keys being in the new context.

Reply:
I thought about that too! You probably mean we use a database for our stuff, then delete the entire database in `Clear()`, or even just start a new one for the next benchmark without clearing the old one?